config/sendMail.js

```diff
@@ -0,0 +1,35 @@
+const nodemailer = require('nodemailer');
+const { emailID, emailPassword, port } = require('./config');
+
+const transporter = nodemailer.createTransport({
+  service: 'gmail',
+  auth: {
+    user: emailID,
+    pass: emailPassword
+  }
+});
+
+const host = `http://localhost:${port}/api/v1`;
+
+module.exports.sendMailForVerifyAccount = async (
+  username,
+  usermail,
+  token,
+  role
+) => {
+  const link = `${host}/auth/verifyAccount?token=${token}&role=${role}`;
```

Comment:
Bhai, there are three types of user. customer, delivery boy and admin. I store customer and delivery boy in USER database and admin in ADMIN database.. So I am sending here role because If i wont, I had to search that user in two database (I had to do two query). If I send it, then I will know, It is from which database and one query needed to find that ID.

Reply:
Okay, great explanation. :D @LetsTrie